include/cling/Interpreter/Interpreter.h

```diff
@@ -126,6 +126,8 @@ namespace cling {
       kExeCompilationError,
       ///\brief The function is not known.
       kExeUnkownFunction,
+      ///\brief The Transaction had no module (probably an error in CodeGen).
+      kExeNoModule,
```

Comment:
Do we really need this state? Isn't this `kExeCompilationError` with the `Transaction` having a `nullptr` module?